pkg/opentelemetry/output.go

```diff
@@ -36,47 +52,198 @@ func New(p output.Params) (*Output, error) {
 
 // Description returns a human-readable description of the output that will be shown in `k6 run`
 func (o *Output) Description() string {
-	return "OpenTelemetry: " + o.config.Address
+	return "opentelemetry: " + o.config.String()
```

Comment:
```suggestion
	return "OpenTelemetry: " + o.config.String()
```
If there isn't a specific reason, I would prefer the CamelCase version. In this way, we are consistent with the ecosystem.

Reply:
My rationale was that this displays in the console first, and having the only word `OpenTelemetry` there sounds inconsistent. 

See:
![image](https://github.com/grafana/xk6-output-opentelemetry/assets/5425600/9702742e-4557-44d0-9003-f46de791971c)

I think that's why most of our outputs have all descriptions lowercased only:

* https://github.com/grafana/k6/blob/master/output/json/json.go#L56
* https://github.com/grafana/k6/blob/master/output/statsd/output.go#L84
* https://github.com/grafana/k6/blob/master/output/csv/output.go#L155

Anyway, I don't have a strong preference here, so just apply your suggestion for now.

Reply:
Definitely better to be consistent with the rest. Sorry for the noise here